### ISSUE TITLE
MediaWiki version 1.29 - wfLoadExtension()

### ISSUE THREAD

**Issue by pedwik** [tessus/mwExtensionMantis#30]
I am now updating a wiki site from version 1.20 to 1.29, and notice there is a new recommended way to load extensions, using a method wfLoadExtension() instead of require_once. But this requires some json files to be included with the extension. Is this something you are planning to do?

**Comment by tessus** [comment]
Yes, it is on my list. But it is not a pressing issue, thus I haven't done anything yet.

The extension works perfectly fine without `wfLoadExtension()`.

**Comment by tessus** [comment]
I've been looking into it, but this might take a while.

I will have to rewrite the entire code to use classes and methods. :-(
